tests/test-cases/Group6-VIC-Machine/6-5-Create-Validation.robot

```diff
@@ -26,10 +26,22 @@ Suggest resources - Invalid target path
     Set Test VCH Name
 
     Log To Console  \nInstalling VCH to test server...
-    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL}/MUCH/DATACENTER --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} ${vicmachinetls}
+    ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL}/MUCH/DATACENTER --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD} ${vicmachinetls}
     Should Contain  ${output}  Suggested datacenters:
     Should Contain  ${output}  vic-machine-linux failed
 
+Create VCH - target thumbprint verification
+    Log To Console  \nRunning vic-machine create - thumbprint verification
+    Set Test Environment Variables
+    # Attempt to cleanup old/canceled tests
+    Run Keyword And Ignore Error  Cleanup Dangling VMs On Test Server
+    Run Keyword And Ignore Error  Cleanup Datastore On Test Server
+    Set Test VCH Name
+
+    Log To Console  \nInstalling VCH to test server...
+    ${output}=  Run  bin/vic-machine-linux create --thumbprint=NOPE --name=${vch-name} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --image-store=%{TEST_DATASTORE} ${vicmachinetls}
```

Comment:
could specify other invalid arguments here (such as invalid image store) to ensure that thumbprint is the first failure,